src/redux/search/searchIndex.js

```diff
@@ -0,0 +1,90 @@
+import React from 'react';
+import { useSelector } from 'react-redux';
+import Spinners from '../../components/spinners';
+import { selectAllJobs } from './searchSlice';
+
+const SearchIndex = () => {
```

Comment:
- [ ] On your Home page, your filter is supposed to filter through the items already on the homepage and not navigate to the category directly?


https://user-images.githubusercontent.com/52934128/194131352-4920c6e2-faa9-4414-80ca-c090ec904f18.mp4

Reply:
Thanks again @topeogunleye for the review, I will restructure my codebase to bear these changes as required herein. This wasn't possible with the present implementation but, I already got a cue on how to make this work. Thanks again for your review.

Reply:
you're welcome